### ISSUE TITLE
Replace bootstrap `alert` classes

### ISSUE THREAD

**Issue by mattcollier** [digitalbazaar/bedrock-angular-authn#4]
https://github.com/digitalbazaar/bedrock-angular-authn/blob/master/login-modal-component.html#L7-L10

We're using bootstrap `alert` classes here.

Material does not appear to have equivalent classes we can use.  Do we want to use custom CSS and if so, where should we put it?

This SO post suggest lifting the `alert` classes out of bootstrap: https://stackoverflow.com/a/40059571

**Comment by dlongley** [comment]
Yeah, I noticed their absence but never got around to solving this issue before.

This is another approach:

https://mdbootstrap.com/angular/advanced/alerts/

Angular-material has "toast": https://material.angularjs.org/latest/demo/toast

Perhaps we can colorize it and do alerts that way, as overlays.

**Comment by mattcollier** [comment]
The `alert` classes are used in the demo warning as well: https://github.com/digitalbazaar/bedrock-angular-demo-warning/blob/master/demo-warning-component.html#L1

**Comment by dlongley** [comment]
I think we should explore creating a simple `br-alert` directive that uses `$mdToast` and angular material's theme provider stuff. A `br-alert` would have a `br-show` attribute for when to show/hide -- and, if we want, an option to add a close button to them that would auto-hide them.

Then we can replace wherever we use bootstrap alert classes with `br-alert`.

**Comment by mattcollier** [comment]
@dlongley sounds like a good thing to experiment with.  Why `br-show` instead of using `ng-if`, `ng-show`  etc?

**Comment by dlongley** [comment]
@mattcollier -- because that's how `$mdToast` works ... you have to call functions to show/hide it, from what I can tell. Probably has to do with animation and positioning.

**Comment by dlongley** [comment]
We should also update `brAlerts` to use this new `br-alert` as well. This should cause alerts to be uniform whether occurring in a form or via some other method.

**Comment by mattcollier** [comment]
@dlongley turns out, that we can use [md-card](https://material.angularjs.org/latest/api/directive/mdCard) to achieve the same effect we had before.

Here's the implementation in [bedrock-angular-demo-warning](https://github.com/digitalbazaar/bedrock-angular-demo-warning/blob/material/demo-warning-component.html)

![md-card-demo-warning](https://user-images.githubusercontent.com/902041/29420105-0c0591f2-833f-11e7-91b1-f5e698888f4c.png)

**Comment by dlongley** [comment]
@mattcollier,

Yeah, I did that before but I'm not convinced it's what we actually want. Maybe we have two different classes of alerts. I am thinking that md-toast will work much better for our brAlerts/brAlertService stuff. It could be that md-card will work ok for static, on screen alerts like our demo warning.

**Comment by mattcollier** [comment]
@dlongley we are in agreement, already implemented this for the static use case: https://github.com/digitalbazaar/bedrock-angular-alert/blob/material/alert-component.js

**Comment by dlongley** [comment]
If we're using `br-alert` for the static stuff, what component will we use for the "toast" stuff? Seems like "alert" is a better name for the toast-like stuff and we need another name for whatever these static things are. Thoughts?

**Comment by mattcollier** [comment]
@dlongley I thought `br-toast` would be an appropriate name for toast things.
